pom.xml

```
@@ -42,7 +42,7 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
 
     <!-- jenkins versions -->
-    <jenkins.version>2.426.3</jenkins.version>
+    <jenkins.version>2.472</jenkins.version>
     <bom.artifactId>bom-2.426.x</bom.artifactId>
     <bom.version>3208.vb_21177d4b_cd9</bom.version>
```

An **optional** suggestion to bring the plugin BOM version closer to the minimum Jenkins version
```suggestion
    <bom.artifactId>bom-2.462.x</bom.artifactId>
    <bom.version>3258.vcdcf15936a_fd</bom.version>
```

This is optional because the existing value of 2.426.x is already reasonably modern.  2.462.x is several months more recent than 2.426.x, but only several months more recent.

---

This comment is not helpful because there is no need to change the BOM version as part of this pull request.  Apologies that I didn't detect that sooner while reviewing.